WcaOnRails/config/locales/en.yml

```diff
@@ -1021,6 +1022,31 @@ en:
         content:
           #context: Yup, nothing to translate here! Just hit "Use the original version"
           '1': "%{current_user_claim}"
+      '9':
+        title: "I am registered for a competition and want to sign up for an additional event, what should I do? Can I bring guests?"
+        #context: Answer to "I want to sign up or an additional event ...?"
+        content:
+          '1': "Please contact the organizer. You can find the contact information on the competition website. If you are unsure where to find it, check the <a href='/competitions'>list of competitions</a>."
+      'regulations1':
```

Comment:
I assume that this `regulations1` does what you want here? I just realized I have no idea if there's any guarantee that our FAQs remain in any particular order. @viroulep, the relevant code is [here](https://github.com/jfly/worldcubeassociation.org/blob/546eeb12deb4bc5732674a560bd5c55bf87e476c/WcaOnRails/app/views/static_pages/faq.html.erb#L35), I believe you wrote this, do you know if we should worry about ordering at all?